kiwi/system/setup.py

```diff
@@ -302,8 +299,10 @@ def setup_users(self):
                 user_id = user.get_id()
                 user_realname = user.get_realname()
                 user_shell = user.get_shell()
-                user_secgroups = user.get_secgroups()
-                user_group = user.get_group()
+                if user.get_groups():
+                    user_groups = user.get_groups().split(',')
+                else:
+                    user_groups = None
```

Comment:
@isbm I tend not to use long oneliners, since IMHO they tend to be difficult to read. However you and @schaefi have a higher experience in python than me, so I am totally fine using such a ternary operator, do we have already an style criteria on how to code this situations?

Just for me to know for the future  and be able to code in a coherent and consistent style.